src/lib/twitter/notifyTwitter.ts

```diff
@@ -0,0 +1,23 @@
+import TwitterAPI from 'twitter-api-v2';
+import { NFTSale } from "lib/marketplaces";
+import axios from 'axios';
+
+export default async function notifyTwitter(twitterClient: TwitterAPI, nftSale: NFTSale) {
+    const hasImage = Boolean(nftSale.nftData?.image);
+    const data = hasImage ? await getImageDataFromUrl(nftSale.nftData?.image as string) : null;
+    const nftName = nftSale.nftData?.name;
+    const text = `${nftName} has just been sold for ${nftSale.getPriceInSOL()} S◎L at ${nftSale.marketplace.name}! #SolanaNFTs #NFTSale`
```

Comment:
You could include the bidding method in the text now 
```ts
  const description = `Sold ${
    nftSale.method === SaleMethod.Bid ? "via bidding " : ""
  }for ${nftSale.getPriceInSOL()} S◎L at ${marketplace.name}`;
```

Reply:
Done!